azure-pipelines.yml

```diff
@@ -162,6 +162,8 @@ jobs:
       ccache --print-config
       ccache --zero-stats
 
+      COMPILER_VER=$(CompilerVer)
+      echo '##vso[task.setvariable variable=CompilerMajorVer]'${COMPILER_VER:0:1}
```

Comment:
This will make `gcc-1` from `gcc-10` when it is released. Not a problem now but will be in future.

Reply:
Changed to `${COMPILER_VER//[^0-9]/}`. Clang 4.0 will be `40` now.

Reply:
Changed to `${COMPILER_VER%.*}`:
```
$ ver=10.0
$ echo ${ver%.*}
10
```